api/agent/call.go

```diff
@@ -331,6 +331,7 @@ func (c *call) AddUserExecutionTime(dur time.Duration) {
 		c.userExecTime = new(time.Duration)
 	}
 	*c.userExecTime += dur
+	c.Model().ExecutionDuration = *c.userExecTime
```

Comment:
Where do you want to consume this value? we have it in the *call object I guess you want to add it to the Call object (naming...) to use it somewhere where the *call object is not available, can you give some example? Can you add a comment to specify why this will be useful?

Reply:
I've updated that to tat effect, but for the record here:

The point of this is to expose the ExecutionDuration to models.Call, since it's that that any CallListener gets to play with. This gives us a better view of the complete lifecycle of an
invocation on the loadbalancer; in particular, we get to actually meter runtime rather than
(placement wait time plus runtime).